index.html

```diff
@@ -59,71 +56,144 @@ <h1 class="header-title">
+              </p>
             </div>
+          </div>
         </div>
       </div>
     </section>
-    <section class="section">
+    <section class="section-benefits">
       <div class="container">
-        <h1 class="section-top">Download the app now.</h1>
-        <h2 class="section-row">Available on your favorite store. Start your premium experience now</h2>
-        <div class="section-buttons">
-          <button class="header-buttons__btn header-buttons__playstore">
-            Playstore
-          </button>
-          <button class="header-buttons__btn header-buttons__appstore">
-            App store
-          </button>
+        <div class="section-benefit__text">
+          <h2 class="section-benefit__dwn">Download the app now.</h2>
+          <h3 class="section-benefit__description">
+            Available on your favorite store. Start your premium experience now
+          </h3>
+          <div class="section-buttons">
+            <button class="header-buttons__btn header-buttons__playstore"><a class="link-btn" href="https://play.google.com/store">
+              Playstore
+            </a></button>
+            <button class="header-buttons__btn header-buttons__appstore"><a class="link-btn" href="https://www.apple.com/ua/app-store/">
+              App store
+            </a></button>
+          </div>
         </div>
       </div>
     </section>
     <footer class="footer">
-      <div class="container">
         <div class="footer-icons">
-          <img class="footer-logo" src="images/Bella Olonje logo 111 1.png" alt="Bella Olonje">
-          <img class="footer-icons__items" src="images/Vector.png" alt="Twitter">
-          <img class="footer-icons__items" src="images/Vector (1).png" alt="Facebook">
-          <img class="footer-icons__items" src="images/Vector (3).png" alt="Instagram">
-          <span class="footer-text">Copywright 2020 Bella Onojie.com</span>
+            <div class="footer-logo">
+              <a href="/"><img
+                src="images/Bella Olonje logo 111 1 (1) (1).png"
+                alt="Bella Olonje"
+              /></a>
+            </div>
+            <div>
+              <a href="https://twitter.com/home?lang=ru">
+                <img
+                class="footer-icon__social"
+                src="images/Vector.png"
+                alt="Twitter"
+              />
+              </a>
+              <a href="https://facebook.com/">
+                <img
+                class="footer-icon__social"
+                src="images/Vector (1).png"
+                alt="Facebook"
+              />
+              </a>
+              <a href="https://www.instagram.com/">
+                <img
+                class="footer-icon__social"
+                src="images/Vector 4.png"
+                alt="Instagram"
+              />
+              </a>
+            </div>
+          <p class="footer-link"><a class="link-item" href="/">Copywright 2020 Bella Onojie.com</a></p>
```

Comment:
Remove class footer-link from paragraph.
And let's change link-item class to the footer-copyright__link